docs_src/nosql_databases/tutorial001.py

```diff
@@ -1,11 +1,10 @@
 from typing import Optional
 
-from fastapi import FastAPI
```

Comment:
We don't want to use isort in this tutorial, because `fastapi` and `pydantic` are really third party imports here.

Reply:
This is fixed